pkg/virt-controller/watch/testing/testing.go

```diff
@@ -42,10 +42,15 @@ func MarkAsNonReady(vmi *v1.VirtualMachineInstance) {
 }
 
 func VirtualMachineFromVMI(name string, vmi *v1.VirtualMachineInstance, started bool) *v1.VirtualMachine {
+	runStrategy := v1.RunStrategyAlways
```

Comment:
how did we miss it until now?
Would you add a word in the commit message why it should go away?

BTW, do we have a unit test for `running`? We should keep it tested once until API v2.

Reply:
> Would you add a word in the commit message why it should go away?

Done :+1: 

> BTW, do we have a unit test for running? We should keep it tested once until API v2.

I definitely agree. I haven't removed any unit tests related to it and I see still many occurrences in unit tests.